cmd/standalone/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.7
+EXPOSE 5775 6831 6832 14267 3000 3001 8080 8081 8082 8083
+
+COPY ./examples/hotrod/hotrod-demo-linux /bin/
+COPY ./cmd/standalone/standalone-linux /bin/
+
+ENTRYPOINT /bin/hotrod-demo-linux all & /bin/standalone-linux --span-storage.type=memory --query.port=3001
```

Comment:
I think you're running it incorrectly. There's no 3001 in our internal build of query + ui.